### ISSUE TITLE
input_default_bindings not working on macOS

### ISSUE THREAD

**Issue by kpj** [jaseg/python-mpv#63]
Hello,

Time for another macOS related issue ( :-[ ). As explained in #61, an extra QT-Window needs to be created for python-mpv to work on macOS.

If I now take the [PyQT embedding](https://github.com/jaseg/python-mpv#pyqt-embedding) example from the README and add `input_default_bindings=True, input_vo_keyboard=True` to the MPV instantiation, I would expect mpv-commands such as `space` (for pausing) or `o` (for time-display) to work.
On linux, it does indeed work.
On macOS however, nothing happens.

Do you have a suggestion why this might be happening? (PyQT not forwarding keypress-events, ...)

**Comment by jaseg** [comment]
Sorry, I have no idea. Maybe focus order, in that on mac your window is not in focus?

**Comment by kpj** [comment]
I investigated this using a custom `keyPressEvent` method in the `QMainWindow` and it does receive the key-events.
I also added custom `@player.on_key_press` bindings to my MPV class, but they were never triggered.

Is there a way of simulating keypresses using python-mpv? That way I could catch key-events in PyQT's `keyPressEvent` method and propagate them manually.
This is of course a very dirty solution, but the best I could think of so far.
